pages/collections/[slug]/manage/access.tsx

```diff
@@ -18,7 +18,7 @@ function CollectionAccess() {
       initialVariables={{ ...queryVars, collectionSlug }}
     >
       {({ data }) => (
-        <CollectionLayout showSidebar data={data?.collection}>
+        <CollectionLayout showSidebar data={data?.collection} useRouteHeader>
```

Comment:
## Child page headers
The problem:
Child page headers can be one of the following:
1. Plain text
2. Text and right hand content (buttons)
3. Included with a model list

Instead of hiding the model list's header, I've created this flag, `useRouteHeader`, to use when a plain text header is needed.